paddle/phi/infermeta/spmd_rules/batch_norm.cc

```diff
@@ -152,7 +152,7 @@ SpmdInfo BatchNormInferSpmd(const DistMetaTensor& x,
       GetDimsMappingForAxes(mean_axes, axis_to_dim_map));
   saved_variance_dist_attr.set_dims_mapping(
       GetDimsMappingForAxes(variance_axes, axis_to_dim_map));
-  reserve_space_dist_attr.set_dims_mapping({-1});
+  reserve_space_dist_attr.set_dims_mapping(std::vector<int64_t>{-1});
```

Comment:
Since developers are still contributing spmd rules, should we publish a standard for spmd rule development on the official website, explaining the recommended usage of `dims_mapping` in order to avoid problems with newly developed rules in the future